inst/builtin/bs5/shiny/_rules.scss

```diff
@@ -115,6 +115,8 @@ $bslib-checkbox-radio-margin-right: 0.35em !default;
     --bslib-dashboard-sidebar-main-bg: var(--#{$prefix}body-bg);
     --bslib-dashboard-main-bg: rgb(247, 247, 247);
     --#{$prefix}card-border-color: var(--bslib-dashboard-border-color-translucent, var(--#{$prefix}border-color-translucent));
+    --bslib-dashboard-card-cap-bg: var(--#{$prefix}body-bg);
```

Comment:
Do you recall the reasoning for changing from `transparent` to `var(--#{$prefix}body-bg)`?

Reply:
The card might be on a colored background -- like our `.bslib-page-dashboard` area.